dids/document.go

```diff
@@ -182,5 +186,5 @@ type VerificationMethod struct {
 	// a value that conforms to the rules in DID Syntax: https://www.w3.org/TR/did-core/#did-syntax
 	Controller string `json:"controller"`
 	// specification reference: https://www.w3.org/TR/did-core/#dfn-publickeyjwk
-	PublicKeyJwk jwk.JWK `json:"publicKeyJwk,omitempty"`
+	PublicKeyJwk *jwk.JWK `json:"publicKeyJwk,omitempty"`
```

Comment:
the pointer question:
- I changed this here because in the `Unmarshal` method I'm validating if this was actually initialized. `PublicKeyJwk` is used when calling several of the methods you called out; I could also dereference
- `jwk.JWK` is a struct of several strings and it is passed around. Passing it as a reference instead of a copy is not completely wrong. 
- I should update to dereferencing instead, because all those methods don't have any checks for nil

Reply:
By that logic, wouldn't you also want `VerificationMethodKeyPair.PrivateKeyJWK` to be a pointer?